### PR TITLE
Update Color foundation page 

### PR DESCRIPTION
**Pull request by amritadesmet**
-Short term text update of Color page to include **definitions for status, sentiment and neutrality** for September Button release

### CODE REVIEW

File: site/docs/foundations/color.mdx [hunk 1]
```diff
@@ -2,10 +2,35 @@
 title: Color
 layout: DetailTechnical
 data:
-  status: In progress
+
 ---
 
-Colors can be used to communicate meaning, suggest a visual hierarchy and help to distinguish our products from others while providing a cohesive accessible experience.
+Colors can be used to communicate meaning, suggest a visual hierarchy, and help distinguish our products from others while providing a cohesive and accessible experience. In the Salt design system, colors are divided into several categories to convey different concepts.
```
Comment (joshwooding) [inline site/docs/foundations/color.mdx:8]: Even though this was from the old copy, I don't think "our" products makes sense here.

```suggestion
Colors can be used to communicate meaning, suggest a visual hierarchy, and help distinguish products from others while providing a cohesive and accessible experience. In the Salt design system, colors are divided into several categories to convey different concepts. 
```

Comment (joshwooding) [reply site/docs/foundations/color.mdx:8]: Also it is worth mentioning that only color can't be used to communicate meaning, should we add colors can be used with text or icons to communicate meaning?

File: site/docs/foundations/color.mdx [hunk 2]
```diff
@@ -2,10 +2,35 @@
...
 layout: DetailTechnical
 data:
-  status: In progress
+
 ---
 
-Colors can be used to communicate meaning, suggest a visual hierarchy and help to distinguish our products from others while providing a cohesive accessible experience.
+Colors can be used to communicate meaning, suggest a visual hierarchy, and help distinguish our products from others while providing a cohesive and accessible experience. In the Salt design system, colors are divided into several categories to convey different concepts. 
+
+The colors referenced on this page are part of the Salt legacy color library. If you are using the Salt Next theme please refer to the separate color palette and listed guidance on the [Theming page](/salt-ds/blob/colorchangesdoc/site/docs/theming/index.mdx).
```
Comment (joshwooding) [inline site/docs/foundations/color.mdx:10]: Should this be in a callout?

Comment (joshwooding) [reply site/docs/foundations/color.mdx:10]: Are we happy to refer to current salt as "legacy"

File: site/docs/foundations/color.mdx [hunk 3]
```diff
@@ -2,10 +2,35 @@
...
 ---
 
-Colors can be used to communicate meaning, suggest a visual hierarchy and help to distinguish our products from others while providing a cohesive accessible experience.
+Colors can be used to communicate meaning, suggest a visual hierarchy, and help distinguish our products from others while providing a cohesive and accessible experience. In the Salt design system, colors are divided into several categories to convey different concepts. 
+
+The colors referenced on this page are part of the Salt legacy color library. If you are using the Salt Next theme please refer to the separate color palette and listed guidance on the [Theming page](/salt-ds/blob/colorchangesdoc/site/docs/theming/index.mdx). 
+
+## Status
+
+Status is a visual indicator or cue that communicates the current state or condition of a system, component or process to the user. Status helps users to understand what’s happening in the interface, whether it be the progress of a task, availability of a feature or the success of an action. It can be used independent of sentiment.
```
Comment (joshwooding) [inline site/docs/foundations/color.mdx:14]:
```suggestion
Status is a visual indicator or cue that communicates the current state or condition of a system, component, or process to the user. It helps users understand what’s happening in the interface, whether it be the progress of a task, the availability of a feature or the success of an action. Status can be used independent of sentiment. 
```
Also "in the interface" doesn't make sense to me

File: site/docs/foundations/color.mdx [hunk 4]
```diff
@@ -2,10 +2,35 @@
...
-Colors can be used to communicate meaning, suggest a visual hierarchy and help to distinguish our products from others while providing a cohesive accessible experience.
+Colors can be used to communicate meaning, suggest a visual hierarchy, and help distinguish our products from others while providing a cohesive and accessible experience. In the Salt design system, colors are divided into several categories to convey different concepts. 
+
+The colors referenced on this page are part of the Salt legacy color library. If you are using the Salt Next theme please refer to the separate color palette and listed guidance on the [Theming page](/salt-ds/blob/colorchangesdoc/site/docs/theming/index.mdx). 
+
+## Status
+
+Status is a visual indicator or cue that communicates the current state or condition of a system, component or process to the user. Status helps users to understand what’s happening in the interface, whether it be the progress of a task, availability of a feature or the success of an action. It can be used independent of sentiment. 
+
+The Salt design system has 4 status types:
```
Comment (joshwooding) [inline site/docs/foundations/color.mdx:16]:
```suggestion
The Salt design system has 4 statuses: 
```
?

File: site/docs/foundations/color.mdx [hunk 5]
```diff
@@ -2,10 +2,35 @@
...
+The Salt design system has 4 status types: 
+
+1. Success = green
+1. Info = accent (blue or teal)
+1. Warning = orange
+1. Error = red
+
+## Sentiment
+
+Sentiment differs from status in that it is the emotional response or attitude that users experience when interacting with a user interface. It encompasses how components and design elements evoke feelings of positivity, negativity, comfort, trust, or frustration in users. It should always communicate important information.
```
[the remark this answers is not in the document]
Comment (amritadesmet) [reply site/docs/foundations/color.mdx:25]: Good point. Discussed this with Paul (10/09) and decided to add short section for neutrality as well.

File: site/docs/foundations/color.mdx [hunk 6]
```diff
@@ -2,10 +2,39 @@
 title: Color
 layout: DetailTechnical
 data:
-  status: In progress
+
 ---
 
-Colors can be used to communicate meaning, suggest a visual hierarchy and help to distinguish our products from others while providing a cohesive accessible experience.
+Colors can be used to communicate meaning, establish visual hierarchy, and differentiate Salt products from others, all while ensuring a cohesive and accessible experience. Colors are divided into several categories to communicate several concepts across the Salt design system.
```
Comment (joshwooding) [inline site/docs/foundations/color.mdx:8]:
```suggestion
Colors can be used to communicate meaning, establish visual hierarchy, and differentiate products, all while ensuring a cohesive and accessible experience. Colors are divided into several categories to communicate several concepts across the Salt design system.
```

Do we have to limit this to Salt products? Also "from others" feels a bit redundant?

**Review by joshwooding — APPROVED** [review]

**Comment by bhoppers2008** [comment]
Is this based on the latest thread? I thought the emotional response line had been removed?

File: site/docs/foundations/color.mdx [hunk 7]
```diff
@@ -2,10 +2,40 @@
 title: Color
 layout: DetailTechnical
 data:
-  status: In progress
 ---
 
-Colors can be used to communicate meaning, suggest a visual hierarchy and help to distinguish our products from others while providing a cohesive accessible experience.
+Colors can be used to communicate meaning, establish visual hierarchy, and differentiate products, all while ensuring a cohesive and accessible experience. Colors are divided into several categories to communicate several concepts across the Salt design system.
+
+The raw color values on this page reference the current Salt theme. If you’re using the Salt Next theme please refer to the separate color palette and listed guidance on the [Themes page](/salt/theming/index).
```
Comment (joshwooding) [inline site/docs/foundations/color.mdx:9]:
```suggestion
The raw color values on this page reference the current Salt theme. If you’re using the Salt Next theme please refer to the separate color palette and listed guidance on the [Themes page](/salt/themes/index).
```

**Review by joshwooding — DISMISSED** [review]

**Review by origami-z — DISMISSED** [review]
Approve in case this needs to go out ASAP.

I'm confused now on what does foundation mean with this change. Status and sentiment color association is implementation not "principle"?

File: site/docs/foundations/color.mdx [hunk 8]
```diff
@@ -2,10 +2,40 @@
...
 ---
 
-Colors can be used to communicate meaning, suggest a visual hierarchy and help to distinguish our products from others while providing a cohesive accessible experience.
+Colors can be used to communicate meaning, establish visual hierarchy, and differentiate products, all while ensuring a cohesive and accessible experience. Colors are divided into several categories to communicate several concepts across the Salt design system.
+
+The raw color values on this page reference the current Salt theme. If you’re using the Salt Next theme please refer to the separate color palette and listed guidance on the [Themes page](/salt/themes/index).
+
+## Status
+
+Status colors help users understand what’s happening, what actions are needed, and whether any issues need to be addressed. They can be used to reflect changes in the state or condition of a system or process.
```
Comment (joshwooding) [inline site/docs/foundations/color.mdx:13]:
```suggestion
Status colors help users understand what’s happening, what actions are needed, and whether any issues need to be addressed. These colors can be used to reflect changes in the state or condition of a system or process.
```

File: site/docs/foundations/color.mdx [hunk 9]
```diff
@@ -2,10 +2,40 @@
...
 ---
 
-Colors can be used to communicate meaning, suggest a visual hierarchy and help to distinguish our products from others while providing a cohesive accessible experience.
+Colors can be used to communicate meaning, establish visual hierarchy, and differentiate products, all while ensuring a cohesive and accessible experience. Colors are divided into several categories to communicate several concepts across the Salt design system.
+
+The raw color values on this page reference the current Salt theme. If you’re using the Salt Next theme please refer to the separate color palette and listed guidance on the [Themes page](/salt/themes/index).
+
+## Status
+
+Status colors help users understand what’s happening, what actions are needed, and whether any issues need to be addressed. These colors can be used to reflect changes in the state or condition of a system or process.
```
Comment (joshwooding) [inline site/docs/foundations/color.mdx:13]:
```suggestion
Status colors are visual indicators that reflect changes in the state or condition of a system or process. These colors help users understand what’s happening, what actions are needed, and whether any issues need to be addressed.
```

File: site/docs/foundations/color.mdx [hunk 10]
```diff
@@ -2,10 +2,40 @@
...
+Sentiment colors guide users through an interface by providing visual cues about potential outcomes. These colors evoke specific feelings such as negativity, cautiousness, comfort, trust or positivity.
+
+The Salt design system includes 5 sentiments with associated colors:
+
+| Sentiment | Color                     | Feeling     |
+| --------- | ------------------------- | ----------- |
+| Negative  | Red                       | Negativity  |
+| Caution   | Orange                    | Carefulness |
+| Neutral   | Gray                      | Comfort     |
+| Accented  | Blue (Teal in Next theme) | Trust       |
```
[the remark this answers is not in the document]
Comment (amritadesmet) [reply site/docs/foundations/color.mdx:35]: Good point, it's misleading - changing this to read better

**Review by joshwooding — DISMISSED** [review]

**Review by joshwooding — DISMISSED** [review]

**Review by joshwooding — APPROVED** [review]

**Review by origami-z — APPROVED** [review]

**Pull request merged**